src/kotlinSessions/SessionTwo.kt

```diff
@@ -0,0 +1,64 @@
+package kotlinSessions
+
+fun main() {
+
+    //---------------------------------Loops----------------------------------
+
+    // a loop is a block which is designed to be of a repeated execution
+    // so instead of saying
+    // println(x + 1)
+    // println(x + 1)
+    // we would commonly say
+    println("Test while loop")
+    var i: Int = 0
+    while (i < 2) {
+        i = i + 1
+        println(i)
+    }
```

Comment:
I think if you start with for loop it will be better for understanding and just put this ex
```.kt
for( i in 1..5 ) {
      println(i)
}
```

Reply:
Alright, I will.